profiles/sssd/switchable-auth

```diff
@@ -0,0 +1,44 @@
+{imply "with-switchable-auth" if "with-switchable-passkey-auth"}
+{imply "with-switchable-auth" if "with-switchable-web-auth"}
+auth required pam_debug.so auth=authinfo_unavail {exclude if "with-switchable-auth"}
+{continue if "with-switchable-auth"}
+auth      [success=done ignore=ignore default=bad]     pam_selinux_permit.so
+auth      required                                     pam_env.so
+auth      required                                     pam_faildelay.so delay=2000000
+auth      required                                     pam_faillock.so preauth silent                         {include if "with-faillock"}
+auth      [default=1 ignore=ignore success=ok]         pam_usertype.so isregular
+auth      [default=1 ignore=ignore success=ok]         pam_localuser.so
+auth      sufficient                                   pam_unix.so nullok
+auth      [default=1 ignore=ignore success=ok]         pam_usertype.so isregular
+auth      sufficient                                   pam_sss.so
+auth      optional                                     pam_gnome_keyring.so only_if=login auto_start          {include if "with-pam-gnome-keyring"}
+auth      required                                     pam_deny.so
+
+account   required                                     pam_access.so                                          {include if "with-pamaccess"}
+account   required                                     pam_faillock.so                                        {include if "with-faillock"}
+account   sufficient                                   pam_systemd_home.so                                    {include if "with-systemd-homed"}
+account   required                                     pam_unix.so
+account   sufficient                                   pam_localuser.so                                       {exclude if "with-files-access-provider"}
+account   sufficient                                   pam_usertype.so issystem
+account   [default=bad success=ok user_unknown=ignore] pam_sss.so
+account   required                                     pam_permit.so
```

Comment:
Wouldn't it be better to include `password-auth`?

Reply:
I installed authselect latest update, modified `gdm-switchable-auth` to include `switchable-auth` and enabled the following `authselect` configuration:
```
Profile ID: sssd
Enabled features:
- with-sudo
- with-smartcard
- with-switchable-auth
- with-switchable-passkey-auth
- with-switchable-web-auth
- with-systemd-homed
```

All authentication mechanisms work and the user is able to change their password on expiration.

Reply:
Ok, I updated it and included system-auth for account, session and password. It is the same as password-auth.